content/post/But_I_only_use_NONMEM.Rmd

```diff
@@ -0,0 +1,142 @@
+---
+title: "'...but I only use NONMEM'"
+author: "Mike K Smith"
+date: 2017-01-20T14:36:00Z
+tags: ["Standards"]
+output: html_document
+---
+
+## Do you want others to use your models?
+
+Well, you ***SAY*** you only use NONMEM (although I don't believe that - see 
+below) but ultimately I'm sure that you'd like other people to use your model 
+and/or the inferences based on it.
+
+The use of modelling and simulation has expanded into quantitative pharmacology,
+pharmacology (Pop PK and PKPD), disease modelling, model-based meta-analysis 
+(MBMA), cost-effectiveness and many more. Many different quantitative 
+disciplines are now involved in building models, evaluating models,
+using models and modelling outputs to interpret data, plan studies or make 
+decisions. For a given model to be useful (and used!) across all of these groups
+we need to find a way that we can express the model such that anybody else that
+wants to use it can do so for their own purposes, whatever those are.
+
+## How do you share models?
+
+### I share via (NMTRAN / other) code
+[A recent survey by the International Society of Pharmacometrics](http://discuss.go-isop.org/t/isop-global-survey-how-do-you-share/657) 
+found that ~75% of respondents primarily use NONMEM as their main tool for 
+analysis. This would support the strategy for sharing NMTRAN models ***within***
+the pharmacometrics community, but what about outside of it?
+
+Within the statistics community there is a growing use of R as a tool for 
+analytical work, sometimes calling other tools e.g. C++, Java, etc. but still,
+within industry, SAS is primarily used as the clinical trial reporting tool of
+choice. 
+
+So if you're interested in getting statisticians to use your model, then
+what language should you choose for sharing? Do you expect them to learn and 
+understand the NMTRAN code? Do you both try to translate to R? What about SAS?
+Is it worthwhile recoding in SAS? Will you learn R or SAS sufficient to be able 
+to share your model?
+
+`<DIGRESSION>`
+*I have been involved in discussions in the past within the statistics community
+about "validation" of R and analysis done with R. The misconception was that 
+analysis performed with R had to be replicated using SAS in order for it to be 
+"validated" and accepted by regulatory agencies. See 
+[here](http://www.slideshare.net/MikeKSmith/r-eliminating-fear-uncertainty-and-doubt)
+for more discussion on this topic. But if we follow that argument through,
+why aren't people insisting that we re-run NONMEM analyses using PROC NLMIXED?*
+`</end DIGRESSION>`
+
+### I share via publications
+Writing down the maths of a model is a great way to share the model between
+quantitative disciplines. It's a universal language and brings with it rigour
+and a lack of ambiguity. ***BUT*** while two modellers can agree on the maths 
```

Comment:
Maths comes with assumptions, and most authors do their best to state these assumptions. Can you imagine the horror when the implemented math is different from the assumptions that the author was making?